databuilder/utils/spark_driver.py

```diff
@@ -9,21 +9,24 @@
 from databuilder.utils.minio_conf import MinioConf
 
 
-def initSparkSession(minio_conf: MinioConf, k8s_hostname: str, k8s_port: int = 6443) -> Tuple[SparkContext, SparkSession]:
+def initSparkSession(minio_conf: MinioConf,
```

Comment:
I'm planning on setting a Kubernetes namespace config, is there a reason we're not doing it here? Is it going to be a parameter?

Reply:
This PR doesn't have any functionality change -- purely types and code format.